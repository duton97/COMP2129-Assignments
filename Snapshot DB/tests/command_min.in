SET a 3 4 5
MIN a
MIN b
BYE
