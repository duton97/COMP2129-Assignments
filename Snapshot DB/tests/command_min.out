> ok

> 3

> no such key

> bye
